config/resnet_test.yaml

```diff
@@ -0,0 +1,15 @@
+gpu_list:
+    - 0
+    - 1
+seed: null
```

Comment:
I'm guessing this is fine, since it works, but why is it `null` here instead of `None`? I wasn't aware of any `null` in python so is this a hydra thing?

Reply:
Oh it's yaml, where there is `null` instead of `None`